src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -52,3 +52,24 @@ export const POPULARITY_WEIGHT: {
   'amazon-sqs': 2,
   splunk: 2,
 } as const;
+
+export const documentIntegrations = {
+  datadog: {
+    slug: 'datadog',
+    name: 'Datadog',
+    author: 'Datadog',
+    docUrl: 'https://www.datadoghq.com/',
+    description:
+      'Quickly discover relationships between production apps and systems performance. Seeing correlations between Sentry events and metrics from infra services like AWS, Elasticsearch, Docker, and Kafka can save time detecting sources of future spikes.',
+    features: [
+      {
+        featureGate: 'data-forwarding',
```

Comment:
@NisanthanNanthakumar  Datadog is actually not a data forwarding integration which would be restricted for the billing plan. Data forwarding plugins trigger on every error while the Datadog integration just uses normal webhooks. Talk to Phillip about what category to use.

Reply:
@scefali The content is just a placeholder. It's in the PR objective.